src/Tickets/Commerce/Settings.php

```diff
@@ -279,6 +271,22 @@ public function get_settings() {
 				],
 				'tooltip_first'   => true,
 			],
+			'tickets-commerce-settings-currency-heading'    => [
+				'type' => 'html',
+				'html' => '<h3>' . __( 'Currency', 'event-tickets' ) . '</h3>',
+			],
+			static::$option_currency_code                   => [
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Currency Code', 'event-tickets' ),
+				'tooltip'         => esc_html__( 'The currency that will be used for Tickets Commerce transactions.', 'event-tickets' ),
+				'default'         => 'USD',
+				'validation_type' => 'options',
+				'options'         => $paypal_currency_code_options,
```

Comment:
I think it's maybe a good option to rename this one? And make it TC related rather than Paypal?